package.json

```diff
@@ -139,6 +139,9 @@
     "test-client": "NODE_ENV=test NODE_PATH=test:client TEST_ROOT=client test/runner.js",
     "test-server": "NODE_ENV=test NODE_PATH=test:server:client TEST_ROOT=server test/runner.js",
     "test-test": "NODE_ENV=test NODE_PATH=test:client TEST_ROOT=test test/runner.js",
+    "test-client:watch": "nodemon -e js,jsx --exec npm run test-client",
```

Comment:
I've added a new issue for the `config` file error here: https://github.com/Automattic/wp-calypso/issues/6605.

As it is a completely unrelated problem I believe it deserves it's own issue